scripts/docker/alpine/3.10/release.Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM alpine:3.10
+
+RUN apk update \
+    && apk add --no-cache --upgrade\
+        bison \
+        build-base \
+        cmake \
+        curl \
+        diffutils \
+        file \
+        git \
+        ninja \
+        ronn \
+        yaml-cpp-dev
+
```

Comment:
Maybe some additional deps like yajl, xerces, augeas would be nice. (If they are available)